source/reference/api/reverse.txt

```diff
@@ -99,3 +97,25 @@ entire sync process to copy the original data.
 To view the mapping direction for the synchronization of the source and
 destination clusters, use the :ref:`progress <c2c-api-progress>`
 endpoint and check the ``directionMapping`` object.
+
+.. _c2c-validate-unique-index:
+
+Validate Unique Indexes
```

Comment:
I might consider moving this to the Requirements section. It might be reasonable to keep it as you have it here in a sub-section, and just move it up to the Requirements. My reasoning is that this information seems like something that users might want to know before they try running the command, but I suppose it might depend on how often we expect users to run into this issue. If not often, the placement you have now I think makes sense.

I'll leave the placement decision up to you.

Reply:
Fixed